mmv1/third_party/terraform/services/container/go/resource_container_node_pool.go.tmpl

```diff
@@ -1412,6 +1412,42 @@ func nodePoolUpdate(d *schema.ResourceData, meta interface{}, nodePoolInfo *Node
 
 	if d.HasChange(prefix + "node_config") {
 
+		if d.HasChange(prefix + "node_config.0.insecure_kubelet_readonly_port_enabled") {
```

Comment:
this is another mistake - I think this is nested in `node_config.0.kubelet_config` already covered, and can be moved